components/tools/OmeroJava/test/integration/chgrp/AnnotationMoveTest.java

```diff
@@ -96,7 +96,9 @@ private void moveImageWithNonSharedAnnotation(String src, String dest,
         assertEquals(iQuery.findAllByQuery(sb.toString(), param).size(),
                 annotationIdsUser1.size());
         n = 0;
-        if (src.equals("rwrw--")) n = annotationIdsUser2.size();
+        if (src.equals("rwrw--") && !dest.equals("rw----")) {
```

Comment:
We do not expect to see another user's annotations once we have moved the image into a private group.